### ISSUE TITLE
Unable to index

### ISSUE THREAD

**Issue by nhsieh** [apnex/vmw-cli#15]
When I run `vmw-cli index vmware-tanzu-kubernetes-grid` (also tried vmware-vsan)
Then I run `vmw-cli find`
I get the following error:
```
[ERROR]: [fileIndex.json] not found, please perform "vmw-cli index" for a solution
[ 0/0 ] entries - filter [ fileName: ]
```

This started happening after June 18, 2020. I am able to log into https://my.vmware.com/ and download the files manually.

The index step output:
```
Updating [fileIndex.json] for all permitted downloads in [vmware-tanzu-kubernetes-grid] ...
Session file[/usr/local/lib/node_modules/vmw-cli/lib/index.json] [65] older than [60] seconds...
Synching delicious cookies from [https://my.vmware.com]
Offering up afforementioned snacks as a sacrifice to [https://auth.vmware.com/oam/server/auth_cred_submit]
Pulling landing index.json [https://my.vmware.com/group/vmware/downloads]
Cookies not tastey enough (error) - retry
Synching delicious cookies from [https://my.vmware.com]
Offering up afforementioned snacks as a sacrifice to [https://auth.vmware.com/oam/server/auth_cred_submit]
Pulling landing index.json [https://my.vmware.com/group/vmware/downloads]
```

Thanks for the help!

**Comment by dashaun** [comment]
This is my first time using the tool.  I am running into the same results on Mac & Ubuntu.

**Comment by apnex** [comment]
Yes this is a known issue - the vmw-cli tool is currently broken due to some 'under-the-hood' changes in the authentication process on my.vmware. I am working on a fix that involves a complete re-write (for the better) of the entire auth, login and indexing function of the tool. Stay tuned for vmw-cli 2.0.

**Comment by Texiwill** [comment]
@apnex please let me know how I can assist or even share my efforts related to Auth due to those under-the-hood changes in authentication. Absolutely willing to collaborate. I did get it working using a very different approach when the normal approach failed.

**Comment by apnex** [comment]
I'm this particular version of vmw-cli now **DEPRECATED**. There have been many major under the hood changes with the website that has necessitated a complete rebuild of this tool. I have been working on this and I'm happy to report that version 2.0 of **vmw-cli** will be available for use in the next week or so.
This will also have added benefits of:
-- Realtime file indexing - always up to date
-- Simplified authentication workflows - leveraging SAML/SSO and will be far more resilient
-- Fewer payload extraction / parsing functions - less brittle to website changes
-- Removing and completely replacing the deprecated request.js module for GOT
-- Potential hooks into other parts of my.vmware.com (possibly licensing, evals etc..)

Thanks all for your patience!
